app/scripts/components/analysis/results/timeseries-data.ts

```diff
@@ -267,17 +267,19 @@ async function requestTimeseries({
       }
     });
 
+    const analysisParams = layersBase.layer.analysis?.params;
```

Comment:
I'd propose to change this to `sourceParams` so it is the same pattern the layer uses - since it is the same type of content.

https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/frontmatter/layer.md